packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -24,18 +22,21 @@ contract ColosseumTest is Colosseum_Initializer {
         vm.prank(asserter);
         pool.deposit{ value: asserter.balance }();
 
-        uint256 interval = oracle.SUBMISSION_INTERVAL();
-        startBlockNumber = oracle.latestBlockNumber();
-        uint256 nextBlockNumber = startBlockNumber + interval;
-
+        // Submit genesis output
+        uint256 nextBlockNumber = oracle.nextBlockNumber();
         // Roll to after the block number we'll submit
         warpToSubmitTime(nextBlockNumber);
         vm.prank(pool.nextValidator());
         oracle.submitL2Output(bytes32(nextBlockNumber), nextBlockNumber, 0, 0, minBond);
 
+        // Submit valid output
+        nextBlockNumber = oracle.nextBlockNumber();
+        warpToSubmitTime(nextBlockNumber);
+        vm.prank(pool.nextValidator());
+        oracle.submitL2Output(bytes32(nextBlockNumber), nextBlockNumber, 0, 0, minBond);
```

Comment:
Just question. Do you have any reason for adding this code block, submitting valid output?
IMHO, the genesis output is already been submitted (which is considered valid), there is no need to submit one more output.

Reply:
That's because `INVALID_BLOCK_NUMBER` in `/packages/contracts/contracts/test/testdata/ColosseumTestData.sol` is `3000`. We use this value and related test data in Colosseum tests.
Otherwise, we can change `INVALID_BLOCK_NUMBER` to a number between 0 and 1800, and remove this code block. In this case we should change related test data (e.g. `PublicInput.number` in line 48 of `ColosseumTestData.sol`, or anywhere else?).

Reply:
Ah, I got it. Thank you for your explanation 🙏